### PR TITLE
Creating personal page for Eric Hudec

### PR DESCRIPTION
**Pull request by ehudec**
I have added a directory for my personal page to "users" and added a link to this page under the Radiation Transport section in the index.html file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,6 +34,7 @@ <h4>Radiation Transport</h4>
 <li><a href="{{ site.url }}users/ravi/">Ravi Kumar</a> (<a href="https://github.com/ravikumar" class="user-mention">@ravi</a>)</li>
 <li><a href="{{ site.url }}users/lamad/">Adam Lam</a> (<a href="https://github.com/adam-lam" class="user-mention">@adam-lam</a>)</li>
 <li><a href="{{ site.url }}users/whitmann/">Nick Whitman</a> (<a href="https://github.com/nickwhitman1993" class="user-mention">@nickwhitman1993</a>)</li>
+<li><a href="https://rtrp.github.io/osu-transport/users/ehudec/">Eric Hudec</a> (<a href="https://github.com/ehudec" class="user-mention">@ehudec</a>)</li>
```
Comment (massimolarsen) [inline index.html]: Could you just change it to 

`href="{{ site.url }}users/ehudec/">`

to fit the style.

Otherwise looks good.

Comment (aaronjamesreynolds) [reply index.html]: Yup, other than that everything looks good to me.

Comment (ehudec) [reply index.html]: The change has been made 👍

**Review by massimolarsen — APPROVED** [review]
Looks good, thanks

**Comment by clemekay** [comment]
Fixed merge conflict, Eric was using a previous version of index.html

**Review by clemekay — APPROVED** [review]
College football??? Let's talk.

**Pull request merged**